app/views/ansible_repository/_repository_form.html.haml

```diff
@@ -48,6 +48,14 @@
           = _("Required")
         %span.help-block{"ng-show" => "angularForm.scm_url.$error.urlValidation"}
           = _("URL must include a protocol (http://, https:// or file://) or be a valid SSH path (user@server:path or ssh://user@address:port/path)")
+    .form-group
+      %label.control-label.col-md-2
+        = _("Verify SSL")
+      .col-md-8
+        %input{:type            => "checkbox",
+               :name            => "verify_ssl",
+               :id              => "verify_ssl",
+               'ng-model'       => "vm.repositoryModel.verify_ssl"}
```

Comment:
Note to anyone else who might not be aware...

If you use `"required" => ""` here, it will make it so that the checkbox is required to be checked...

This was not clear to be at the beginning, so I spent a few hours trying to figure out what else I had done wrong until I realized that modifier was the problem.